src/smif/data_layer/data_handle.py

```diff
@@ -357,6 +412,31 @@ def get_results(self, output_name, model_name=None, modelset_iteration=None,
             decision_iteration
         )
 
+    @property
+    def interventions(self):
+        """A list of all interventions from the current model
+        """
+        return self._get_all_interventions()
```

Comment:
Should this be a call to `self._store.read_interventions(self._model_name)` instead of referring to interventions on the model object? If so, can we delete `_get_all_interventions` below?

Reply:
Yes, I've now removed this method. The `get_current_interventions` method should provide the required functionality for the sector models.